code/AssetLib/FBX/FBXConverter.cpp

```diff
@@ -79,7 +79,7 @@ using namespace Util;
 
 #define MAGIC_NODE_TAG "_$AssimpFbx$"
 
-#define CONVERT_FBX_TIME(time) static_cast<double>(time) / 46186158000LL
+#define CONVERT_FBX_TIME(time) (static_cast<double>(time) * 1000.0 / 46186158000LL)
```

Comment:
Shouldn't you also change `anim->mTicksPerSecond = anim_fps;` to `anim->mTicksPerSecond = 1000.0;` ?
The documentation mentions that animation times are expressed in ticks. To convert from ticks to seconds, a division by mTicksPerSecond is required.